fastify.js

```diff
@@ -114,9 +114,9 @@ function build (options) {
     server.on('clientError', handleClientError)
   }
 
-  // JSON body limit option
-  validateBodyLimitOption(options.jsonBodyLimit)
-  fastify._jsonBodyLimit = options.jsonBodyLimit || DEFAULT_JSON_BODY_LIMIT
+  // body limit option
+  validateBodyLimitOption(options.bodyLimit)
+  fastify._bodyLimit = options.bodyLimit || DEFAULT_BODY_LIMIT
```

Comment:
Should we allow 0 to indicate "no limit" ?

Reply:
Allow no limit means create the security issue. I don't agree.
The user can configure it with a big value anyway.

Reply:
That should be up to the user. But I would choose `-1` to remove the limit. It could be that someone would want to use `0` to restrict receiving post bodies altogether.

Reply:
-1 fine for me.

Reply:
Why would a user do this?

Reply:
No sorry, my badf.

If you need to parse a *very* large body you should use stream with a custom parser. Let's keep it.

Reply:
Using `Number.MAX_SAFE_INTEGER` would set the limit to about `8 PB`. I'd say that's pretty close to having no upper limit.